src/app.scss

```diff
@@ -97,3 +97,7 @@ button:focus:not(:focus-visible) {
     font-size: 1.5rem;
   }
 }
+
+main {
+  padding-bottom: 2.3rem;
+}
```

Comment:
Defining the margins in the `body` on line 17 should correct this. So replace `margin: 0 auto;` with:
```
margin-left: auto;
margin-right: auto;
margin-top: auto;
margin-bottom: 2rem;